Authentication/Login/View/FacebookLoginConfiguration.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  FacebookLoginConfiguration.swift
+//  Authentication
+//
+//  Created by Gustavo Cairo on 1/12/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import FacebookCore
+import FacebookLogin
+
+public final class FacebookLoginConfiguration: LoginProviderConfiguration {
+    
+    public let readPermissions: [ReadPermission]
+    public let publishPermissions: [PublishPermission]
+    public let defaultAudience: LoginDefaultAudience
+    
+    /**
+        Initializes a new instance of FacebookLoginConfiguration.
+     
+        By default, read permissions are included, but not publish permissions.
```

Comment:
I don't quite understand this.
First, I think it's best to specify which read permissions are included.
But what I don't get is the part of publish replacing reading.